src/tqec/display.py

```diff
@@ -35,13 +36,15 @@ def display_templates_svg(
     the templates, it can be very slow for large templates and results in huge
     svg files.
 
-    :param templates: the TemplateOrchestrator instance to display.
-    :param write_svg_file: the path to the SVG file to write.
-    :param canvas_height: the height of the canvas in pixels to draw on.
-    :param plaquette_indices: the plaquette indices that are forwarded to the
-        call to template.instantiate to get the actual template representation.
+    Args:
+        templates: the TemplateOrchestrator instance to display.
+        write_svg_file: the path to the SVG file to write.
+        canvas_height: the height of the canvas in pixels to draw on.
+        *plaquette_indices: the plaquette indices that are forwarded to the call
+            to ``template.instantiate`` to get the actual template representation.
```

Comment:
Are the double-backticks needed? Isn't a simple backtick enough?

Reply:
Yes, I think both [single](https://google.github.io/styleguide/pyguide.html#3821-test-modules) and [double](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) backticks should work. I will change this into single backtick for simplicity.

Reply:
Seems like they are not equivalent: https://stackoverflow.com/a/57302493/4810787.

Reply:
I wasn't aware of that before. So it appears safer to use double backticks for inline code snippets, although single backticks should rarely cause any issues.

Reply:
And I think we can keep on using single backtick.